tests/test_rawpy.py

```diff
@@ -1,48 +1,97 @@
 """Tests for rawpy plugin
 """
 
-import rawpy
 import imageio.v3 as iio
+
+import pytest
+import rawpy
 import numpy as np
 
 
-def test_nef_local(test_images):
+@pytest.mark.parametrize(
+    "im_in",
+    [
+       ("Nikon_uncompressed.nef"),
+       ("Blackmagic.dng"), 
+    ],
+)
+def test_im_remote(test_images, im_in):
     """Test for reading .nef file from .test_images dir.
     """
     # Construct image path
-    im_path = test_images / "infrared.nef"
+    im_path = test_images / im_in
 
     # Test if plugin's content mathces rawpy content
     actual = iio.imread(im_path, index=..., plugin="rawpy")
     expected = rawpy.imread(str(im_path)).postprocess()
     assert np.allclose(actual, expected)
 
 
+@pytest.mark.parametrize(
+    "im_in",
+    [
+       ("camera.png"),
+       ("bricks.jpg"), 
+    ],
+)
```

Comment:
I don't think we need to parametrize a failure test. One image enough to hit the code-path I suspect?

Reply:
The test is removed as per our last discussion. Thanks for the review